doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -81,10 +85,10 @@ To work with the nRF54H20 DK, follow the instructions in the next sections to in
 
 .. _ug_nrf54h20_install_toolchain:
 
-Installing the |NCS| v2.7.99-cs2 and its toolchain
+Installing the |NCS| v2.8.0 and its toolchain
```

Comment:
Good to add the |release| shortcut, which gets updated to the current release.
Also, for below cases.